src/App/Fossa/ManualDeps.hs

```diff
@@ -165,21 +166,22 @@ scanAndUpload ::
   Flag AllowNativeLicenseScan ->
   m (NonEmpty Locator)
 scanAndUpload root vdeps allowNative = do
+  org <- getOrganization
   archiveOrCLI <-
     if fromFlag AllowNativeLicenseScan allowNative
       then do
-        doNative <- orgDoLocalLicenseScan <$> getOrganization
+        let doNative = orgDoLocalLicenseScan org
         if doNative
           then pure CLILicenseScan
           else -- If they've selected native scanning, but the server doesn't support it,
           -- we should let them know.
           -- TODO: Add a --forbid-archive-upload CLI flag
             logWarn "Server does not support native license scanning" $> ArchiveUpload
       else pure ArchiveUpload
-
+  let vendoredDependencyScanMode = if (orgSupportsAnalyzedRevisionsQuery org) then VendoredDependencyScanModeSkipPreviouslyScanned else VendoredDependencyScanModeSkippingNotSupported
```

Comment:
[fmt]: I am legitimately shocked that the formatter did not break this line up automatically.  Please break this line up.